.github/workflows/new_issues.yml

```diff
@@ -18,3 +18,71 @@ jobs:
           column: Backlog
           # TODO: Switch to GITHUB_TOKEN if the project is moved to a repo
           repo-token: ${{ secrets.ACCESS_TOKEN }}
+
+  add_issue_to_project:
```

Comment:
Does this need the priority field which is set by the `set_issue_priority_field` job? Because if so, [`needs`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds) might be needed here.

Reply:
I think it's the other way around, `set_issue_priority_field` needs `add_issue_to_project`. Thank you for suggestion, I'll add it.